src/Lykke.Service.PayInternal/Rabbit/Publishers/PaymentRequestPublisher.cs

```diff
@@ -78,7 +78,8 @@ public void Stop()
         
         public async Task PublishAsync(PaymentRequestDetailsMessage message)
         {
-            await _log.WriteInfoAsync(nameof(PaymentRequestPublisher), nameof(PublishAsync), message.ToJson());
+            await _log.WriteInfoAsync(nameof(PaymentRequestPublisher), nameof(PublishAsync), message.ToJson(),
```

Comment:
save here context not as json
add com text .. $"message: {message.ToJson()}"
mot need parce this in ELK